checkUrlExists/lambda/index.js

```diff
@@ -31,7 +31,11 @@ async function checkUrl(url) {
 }
 
 async function run(url) {
-  return curly.head(url);
+  const curlOpts = {
+    SSL_VERIFYPEER: false,
```

Comment:
Why did these two options get added? They seem dangerous, although I suppose only the lambda would compromised.

Reply:
These were to allow the lambda to work with HTTPS URLs; @garyluu would you please elaborate?

Reply:
I tried running it locally, with node, slightly tweaking the code to do so, and it worked for me (on a Mac). Which does tend to imply something is off with the environment or how we build for the environment that it's deployed on.

If it worked with request/https library, then that implies there's nothing wrong with the environment in the lambda env.

Lots of guesses on my part.

Reply:
Added cert file which seems to have enabled it to work